extensions/amp-story/1.0/amp-story.js

```diff
@@ -775,6 +776,11 @@ export class AmpStory extends AMP.BaseElement {
 
     this.getViewport().onResize(debounce(this.win, () => this.onResize(), 300));
     this.installGestureRecognizers_();
+
+    // Install parallax handlers if opt-in is detected on a mobile environment
+    if (!this.isDesktop_() && this.element.hasAttribute('parallax-fx')) {
+      installParallaxFx(this.win, this.vsync_, this.element);
+    }
```

Comment:
This block currently lives in `initializeListeners_` which is called by `buildCallback`. If I understand correctly, it'll start trying to animate the layers, before they're even visible, actually delaying the initial render.
Could we move this to the very end of the `layoutCallback`, once the story is rendered? Happy to send you a code pointer, but basically I think we could start the parallax effects at the same moment we call `this.initializeLiveStory_();`

Reply:
+1 on my earlier comment, is there any reason why it should be done before the story is rendered, potentially affecting the large contentful paint?
Not really cool to resolve the conversation without any comment :( If you disagree I'm open to discussing this, I might be missing something.

Reply:
Hey Sorry! I think there's a commit missing that would've fixed this and some of @sparhami 's comments, I'll re-do the changes from that now! :)